flow-nodes/api-builder-plugin-format-date/src/flow-nodes.yml

```diff
@@ -5,32 +5,35 @@ flow-nodes:
     description: Example flow-node that formats Date. Uses the popular Moment.js library.
     category: general
     methods:
-      formatDate:
+      formatDate: # This same key is used in `actions.js`
         name: Format Date
         description: Formats a date. See Moment.js format() method.
         parameters:
           date:
+            name: Date
             description: The date to be formatted.
             required: true
             initialType: string
             schema:
               type: string
           format:
+            name: Format
             description: The desired format. Supports all Moment.js supported formats.
             required: true
             initialType: string
             schema:
               type: string
-        outputs:
-          # success output(s) should be first
-          next:
-            name: Next
-            context: $.date
-            schema:
-              type: string
-          # error output(s) should be last
-          error:
-            name: Error
-            context: $.error
-            schema:
-              type: string
+        # "Next" (next) output
+        returns:
+          name: Next
+          description: The operation was successful.
+          context: $.date
+          schema:
+            type: string
+        # "Error" (error) output
+        throws:
+          name: Error
+          description: An unexpected error was encountered.
+          context: $.error
+          schema:
+            type: string
```

Comment:
This is going to be object most of the time, and all of the time if we can guarantee that `moment` always throws an error object

Reply:
What about any module that `moment` might be using?  Again, an impossible ask.  Imo, we should describe the Error in schema.  If it happens to be something other than Error then that is unfortunate.  If we care, then we should test for it in the action wrapper and wrap it in an Error object.  Either that, or we force every plugin to do that.

For base64, I used:
```yaml
            name: Error
            description: An unexpected error was encountered.
            context: $.error
            schema:
              type: object
              properties:
                message:
                  type: string
```

Reply:
`What about any module that moment might be using?`
Any output of dependencies that are inherited and returned directly **become** moment's interface.

Not saying that we need to meticulously track them down, you're not wrong there. We can make a good assumption as to what the error will be, so what you have is also fine, it's only used as documentation after all too.
But if we know cases that it definitely can be different, and when it will be, we should document it.

Reply:
Leave it as it is according to team's decision.